bin/generate-aws-route-table-public

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+main() {
+  cat <<EOF
+# WARNING: this file is generated!
```

Comment:
Would it be useful to also specify that it's this script that generated the file?